src/hotspot/share/opto/cfgnode.hpp

```diff
@@ -101,7 +101,9 @@ class RegionNode : public Node {
   virtual Node* Identity(PhaseGVN* phase);
   virtual Node* Ideal(PhaseGVN* phase, bool can_reshape);
   virtual const RegMask &out_RegMask() const;
-  bool try_clean_mem_phi(PhaseGVN* phase);
+  bool try_clean_mem_phi(PhaseGVN *phase);
+  bool is_self_loop(Node* n, PhaseGVN *phase);
+  bool try_phi_disintegration(PhaseGVN *phase);
```

Comment:
Why these changes and where new definitions?

Reply:
Definitions were removed earlier declarations not needed anymore. Thanks!